README.md

```diff
@@ -161,23 +169,25 @@ public class OutFakeApi extends PureeBufferedOutput {
 }
 ```
 
+Puree stores protobuf log entries interally as byte arrays containing the serialized protobuf. Since the internal storage is agnostic to the protobuf message class itself, if you want to repopulate the protobuf message class, you need to do so in an output plugin. See [demo/OutBufferedProtobufLogcat.java](demo/src/main/java/com/example/puree/logs/plugins/OutBufferedProtobufLogcat.java) for an example.
+
 ### Definition of Filters
 
-If you need to add common params to each logs, you can use `PureeFilter`:
+If you need to add common params to each logs, you can use `PureeJsonFilter` or `PureeProtobufFilter`:
 
 ```java
-public class AddEventTimeFilter implements PureeFilter {
+public class AddEventTimeFilter implements PureeJsonFilter {
```

Comment:
It looks breaking change. Should we remain original name for developers using this Library?

Reply:
It's definitely a breaking change. I can switch all the *Json* classes back to their original names, although it might be confusing for new users. Good point, though. This would make migration easier.

Reply:
IMO, we should write a migration guide document from 4.x to 5.x if we make breaking change.

Reply:
@rkowase I updated the README in this pull request with descriptions and explanation. I don't think we need to write a migration guide since we don't even know who uses this library.

Reply:
I renamed those classes back to their original names.